dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/openapi/Property.java

```diff
@@ -142,16 +142,16 @@ private static List<Property> propertiesIn(Class<?> object) {
         .filter(f -> includeByDefault || isExplicitlyIncluded(f))
         .forEach(addField);
     methodsIn(object)
-        .filter(method -> Property.isGetter(method) || Property.isSetter(method))
         .filter(Property::isExplicitlyIncluded)
+        .filter(Property::isAccessor)
         .filter(method -> !ignoredFields.contains(getJavaPropertyName(method)))
         .forEach(
             method ->
                 addMethod.accept(
                     method, fieldsByJavaPropertyName.get(getJavaPropertyName(method))));
     if (properties.isEmpty() || includeByDefault) {
       methodsIn(object)
-          .filter(Property::isGetter)
+          .filter(Property::isAccessor)
```

Comment:
@cjmamo Please have a look at this change again in the context of your use case. 

I checked back with the changes you had made a while back to the properties selection logic. Back then you did also consider both getters and setters and I think I changed that later to just getters. As far as I can recall I did that thinking setters were unnecessary. But that is not the case. Sometimes we use setters to allow the property input to be another type or to be given using an alias and similar.